docs/framework/network-programming/managing-connections.md

```diff
@@ -22,6 +22,9 @@ ms.assetid: 9b3d3de7-189f-4f7d-81ae-9c29c441aaaa
 # Managing Connections
 
 Applications that use HTTP to connect to data resources can use the .NET Framework's <xref:System.Net.ServicePoint> and <xref:System.Net.ServicePointManager> classes to manage connections to the Internet and to help them achieve optimum scale and performance.  
+
+> [!NOTE]
+> `ServicePoint` and `ServicePointManager` are considered legacy and and exist only as stubs in .NET Core, .NET 5, and later versions. Very little is used by any of the networking stack. They are referenced from `HttpWebRequest` (also legacy) in only a few places, in an attempt to respect some global settings. `ServicePoint` doesn't track anything about current connections used by `HttpClient` (or `HttpWebRequest`, which layers on top of it) and is hardcoded to return 0 for <xref:System.Net.ServicePoint.CurrentConnections%2A>.
```

Comment:
```suggestion
> `ServicePoint` and `ServicePointManager` are considered legacy on .NET Core, .NET 5, and later versions. Most of their properties and methods are not implemented on these platforms. When they are implemented, they don't affect or track anything about `HttpClient` networking APIs.
```

Reply:
Looks good, I just changed "platforms" to "versions", for consistency with the preceding "and later versions" and to reserve "platform" for references to [the OS](https://docs.microsoft.com/en-us/dotnet/standard/glossary#platform).